Makefile

```diff
@@ -0,0 +1,135 @@
+MAKEFLAGS += --no-print-directory
+SOURCES = packages
+DIR = $(shell pwd)
+
+########################################################################################################################
+#
+# HELP
+#
+########################################################################################################################
+
+# COLORS
+RED    = $(shell printf "\33[31m")
```

Comment:
no need for makefile, you can remove it and use lerna stream cli

Reply:
will be removed with webpack structure change